build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -41,7 +41,15 @@ set -xeu -o errtrace
 
 this_dir="$(cd $(dirname $0) && pwd)"
 script_name="$(basename $0)"
-repo_root="$(cd "${this_dir}" && git rev-parse --show-toplevel)"
+
+# We avoid using git rev-parse --show-toplevel here since some filesystem
+# permissions may trigger git errors/warnings when used via docker
+repo_root=$this_dir
+while [ ! -e "$repo_root/.git" ]; do
+  repo_root=$(dirname $repo_root)
+  if [ "$repo_root" = "" ]; then break; fi
```

Comment:
dirname "/" == "/"

(compare against "/")

Reply:
how does this look now